src/main/webapp/assets/modules/common/model/challengeRegistrant.js

```diff
@@ -148,7 +148,7 @@ techlooper.filter("challengeRegistrant", function (apiService, $rootScope, jsonV
           break;
         }
       }
-      //console.log(registrant);
+      registrant.remainingPhaseItems.length && (registrant.selectedPhaseItem = registrant.remainingPhaseItems[0]);
```

Comment:
Line is too long.<br>Expected an assignment or function call and instead saw an expression.